myTinyURL/convert/convert.py

```diff
@@ -4,26 +4,60 @@
 import random, string
 
 class Convert:
+    """This class is the implementation for url conversion
+    """
+
+    # the length of short_id
     CONST_LENGTH = 6
-    def long2short(httpurl):
-        short_id = Convert.gen()
-        record = URL(httpurl=httpurl, short_id=short_id)
+
+    def long2short(http_url):
+        """Convert given long url to a short url
+
+        Call gen_shortid generate a random short id.
+        Save in database.
+        Using SITE_URL generate response
+
+        Args:
+            http_url: a url we want to convert
```

Comment:
Let's put the type of input and response of this function. 
Putting type information in function comments are critical for python program.